server/spec/hypervisor_check_in_spec.rb

```diff
@@ -902,4 +902,35 @@ def create_virtwho_client(user)
     user = user_client(owner, random_string("user"))
     job_detail = async_update_hypervisor(owner, user, 'hypervisor.bind.com', hypervisor_id, ["blah"])
   end
+
+ it 'should be able to update a consumer and check it in as a hypervisor' do
```

Comment:
I would suggest changing this to "should be able to update a consumer with a hypervisor_id, and check it in as a hypervisor concurrently".